packages/react/src/components/Accordion/index.test.tsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Accordion, AccordionTrigger, AccordionContent } from './';
+import axe from '../../axe';
+
+beforeEach(() => {
+  window.matchMedia = jest.fn().mockReturnValue({ matches: false });
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+const renderDefaultAccordion = () => {
+  return render(
+    <Accordion>
+      <AccordionTrigger>Trigger</AccordionTrigger>
+      <AccordionContent>Content</AccordionContent>
+    </Accordion>
+  );
+};
+
+test('should render without errors', () => {
+  renderDefaultAccordion();
+
+  expect(screen.getAllByRole('generic')[1]).toBeInTheDocument(); // Accordion container
+});
+
+test('should pass through props', () => {
+  render(
+    <Accordion role="region">
+      <AccordionTrigger>Trigger</AccordionTrigger>
+      <AccordionContent>Content</AccordionContent>
+    </Accordion>
+  );
+
+  expect(screen.getByRole('region')).toBeInTheDocument();
```

Comment:
This test is a bit different from before since it doesn't validate that the accordion component is a region.